README.md

```diff
@@ -1,66 +1,65 @@
-gRPC-web, and Connect's own protocol.
+Connect is a family of libraries for building and consuming APIs on 
+different languages and platforms. 
 
-The [Connect protocol](https://connect.build/docs/protocol/) is a simple,
-POST-only protocol that works over HTTP/1.1 or HTTP/2. It supports
-server-streaming methods just like gRPC-Web, but is easy to debug in the
-network inspector. Calling a Connect API is easy enough just with the fetch
-API. Try it with our live demo:
+With Connect, you define your schema first:
 
-```ts
-const res = await fetch("https://demo.connect.build/buf.connect.demo.eliza.v1.ElizaService/Say", {
-  method: "POST",
-  headers: {"content-type": "application/json"},
-  body: `{"sentence": "I feel happy."}`
-});
-const answer = await res.json();
-console.log(answer);
-// {sentence: 'When you feel happy, what do you do?'}
+```
+service ElizaService {
+  rpc Say(SayRequest) returns (SayResponse) {}
+}
 ```
 
-Using the client generated by Connect-Web, the same call becomes quite a bit 
-simpler:
+With the magic of code generation, this schema produces servers and clients:
 
 ```ts
 const answer = await eliza.say({sentence: "I feel happy."});
 console.log(answer);
 // {sentence: 'When you feel happy, what do you do?'}
 ```
 
-To get started, head over to the [docs](https://connect.build/docs/web/getting-started)
-for a tutorial. You will also find API documentation and best practices there.
-For using connect-web with your favorite framework, take a look at 
-[connect-web-integration](https://github.com/bufbuild/connect-web-integration).
-
-
-## Supported Browsers
+Unlike REST, the RPCs you use with Connect are typesafe end to end, but they are 
+regular HTTP under the hood. You can see all requests in the network inspector,
+and you can `curl` them if you want:
 
-Connect-Web supports all modern web browsers that implement the widely
-available [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
-and the [Encoding API](https://developer.mozilla.org/en-US/docs/Web/API/Encoding_API).
-The library and the generated code are compatible with ES2017 and TypeScript 4.1.
+```shell
+curl \
+    --header 'Content-Type: application/json' \
+    --data '{"sentence": "I feel happy."}' \
+    https://demo.connect.build/buf.connect.demo.eliza.v1.ElizaService/Say
+```
 
+With Connect for ECMAScript, you can spin up a service in Node.js and call it
+from the web, the terminal, or native mobile clients. Under the hood, it uses 
+[Protocol Buffers](https://github.com/bufbuild/protobuf-es) for the schema, and 
+implements RPC (remote procedure calls) with three protocols: The widely available 
+gRPC and gRPC-web, and Connect's [own protocol](https://connect.build/docs/protocol/), 
+optimized for the web. This gives you unparalleled interoperability with type-safety end 
+to end.
 
-## Connect for Node.js (Preview)
+To get started, head over to the [docs](https://connect.build/docs/web/getting-started)
```

Comment:
Should we link directly to the `web` docs?  Maybe we should have two links here for Node and Web?